codegenerator/cli/templates/dynamic/codegen/src/Types.res.hbs

```diff
@@ -123,34 +123,46 @@ module Block = {
   @get
   external getNumber: t => int = "number"
 
+  @get
+  external getInternalNumber: Internal.eventBlock => int = "number"
+
```

Comment:
Should these not be in the "Internal" module?

Reply:
No, since they are codegened. Ideally, we should not need them.